crates/miden-proving-service-client/src/lib.rs

```diff
@@ -13,12 +13,7 @@ extern crate std;
 use thiserror::Error;
 
 #[cfg(feature = "tx-prover")]
-pub mod generated;
-
-#[cfg(feature = "tx-prover")]
-mod prover;
-#[cfg(feature = "tx-prover")]
-pub use prover::RemoteTransactionProver;
+pub mod prover;
```

Comment:
nit: should we call the module `tx_prover`?

Reply:
Yes.